tools/aa/commands/InitCommand.java

```diff
@@ -95,11 +97,41 @@ public boolean run(String[] args) {
     return true;
   }
 
+  private boolean processArgs(String[] args) {
+    boolean isAaSetUp = args[0].equals("aa");
+    if (isAaSetUp) {
+      switch (args.length) {
+        case 1:
+          System.err.println(
+              RED_ERROR
+                  + "Invalid usage. \n"
+                  + "Please use \"aa init <base_path>\" command to init a base folder.");
+          return false;
+        case 2:
+          System.err.println(RED_ERROR + "Missing base_path");
+          return false;
+        default:
+          basePath = args[2];
+          break;
+      }
+    } else {
+      switch (args.length) {
```

Comment:
Instead of duplicating, how about you remove the first element if needed, using:
`Arrays.copyOfRange(Object[] src, int from, int to)`
Then, you can use the same code?

Reply:
- [x] Done